README.md

```diff
@@ -142,25 +144,34 @@ Collecting metrics of the above tests as well as testing recovery of failures. E
 ## Anatomy Of A Test Run
 There are several conditional checks (is this an upgrade test, is this a dry-run) that may impact what stages an OSDe2e run may contain, but the most complicated is an upgrade test:
 
-
 1. Load Config
-2. Provision Cluster
+2. Provision Cluster (If Cluster ID or Kubeconfig not provided)
 3. Verify Cluster Integrity
 4. Run Tests (pre-upgrade)
-5. Capture / Upload logs to GCS
+5. Capture logs, metrics, and metadata
 6. Upgrade Cluster
 7. Verify Cluster Integrity
 8. Run Tests (post-upgrade)
-9. Capture / Upload logs to GCS
+9. Capture logs, metrics, and metadata 
+10. CI-Operator uploads artifacts to GCS
```

Comment:
This isn't part of things AFAIK -- however prow is configured just automagically does this.